Project.toml

```diff
@@ -67,7 +67,7 @@ ImageFiltering = "0.6"
 ImageMagick = "1.1"
 ImageTransformations = "0.8"
 KernelDensity = "0.5.1"
-MakieLayout = "0.4, 0.5"
+MakieLayout = "=0.5.0"
```

Comment:
Semver will only allow 0.5.x, so we should be able to relax this to "0.5"?

Reply:
Well, since we have such a tight integration with MakieLayout, and small visual changes are not considered breaking in MakieLayout, I decided we must fix the exakt version to not run into failing tests just because of a minor change